### ISSUE TITLE
Drag and drop folders

### ISSUE THREAD

**Issue by Yashimaru-snm** [PJDude/dude#10]
Hi very useful program. Is it possible to add drag and drop for folders?

**Comment by PJDude** [comment]
looks doable, give me some time

**Comment by PJDude** [comment]
Hello, can You try latest pre-release ([v1.0429.542](https://github.com/PJDude/dude/releases/tag/v1.0429.542)) in Your environment ? Minor details need to be polished but, main window and scan dialog should react for dragged/dropped folder/file(s).

**Comment by Yashimaru-snm** [comment]
Yes, drag and drop works very well in main window and scan dialog. I had no problem. Really thank you very much for this program. Greetings
